fullscreen/trusted-click.js

```diff
@@ -14,6 +14,7 @@ function trusted_click(test, callback, container)
         container.removeChild(button);
     });
     container.appendChild(button);
+    test_automation.click(button);
```

Comment:
(yes, I realise this will break every other manual test using `trusted-click.js` for now, and it shouldn't land as is)